relation_engine_server/utils/arango_client.py

```diff
@@ -9,6 +9,20 @@
 from relation_engine_server.utils.config import get_config
 
 _CONF = get_config()
+adb_url = _CONF["api_url"]
```

Comment:
`adb_url` is already defined on line 31, and you've only used this value of `adb_url` once, on line 19. I would get rid of this variable.